x11/xfce4-dashboard/files/pkg-message.in

```
@@ -0,0 +1,9 @@
+[
+{
+	message: <<EOM
+The 'gdk' backend is not avaiable, so do not try to define
+the XFDASHBOARD_BACKEND environment variable, because value
+will always be 'x11'.
```

If I get this message correctly an invalid value (like 'xyz', and 'gdk' being only a special case) will be silently ignored and thee default of 'x11' will be used. In this case this pkg-message is unnecessary. So I'm removing this

If I'm wrong and the special 'gdk' case causes bad behavior I'll evaluate reinstating this pkg-message.

---

In fact Xfdashboard provides 2 backends:
- x11
- gdk

But in our graphics/clutter port, the gdb backend is missing, that's why I create pkg-message, if somebody reads documentation and decide to try one or the other.

Backends  checked at build time, so you are right.

---

I see. Fact is I usually try to avoid spamming users with many pkg--messages. Personally I find it annoying to always have the same information forced on me.

Looks like ports miss some place where to have information to be shown only on demand.

I'm unsure if UPDATING is the right place for this.

---

For UPDATING I'm not sure. We can ignore this message for simplicity.

In fact, this port needs more feedback especially for desaturation effect (post-patch target). It is based on my experience on 2 different systems and with 2 different graphics cards, I have same behavior with default settings, icons are completely black.

---

I'll leave things as is without a message for now.

ports can be modified later anyway.

For desaturation effect I'm taking the number you proposed since I don't know better. We'll see what will happen.